crates/uv-python/src/managed.rs

```diff
@@ -65,7 +65,7 @@ pub struct ManagedPythonInstallations {
 
 impl ManagedPythonInstallations {
     /// A directory for Python installations at `root`.
-    fn from_path(root: impl Into<PathBuf>) -> Self {
+    pub fn from_path(root: impl Into<PathBuf>) -> Self {
```

Comment:
Instead, I'd update `from_settings` to take an `Option<PathBuf>` like we do for the `Cache` struct

https://github.com/astral-sh/uv/blob/d0dda3798d2514350ab29e4dc3870cd56479a713/crates/uv-cache/src/cli.rs#L42